packages/fuel-indexer-tests/tests/e2e/indexing_postgres.rs

```diff
@@ -108,6 +109,7 @@ async fn test_can_trigger_and_index_callreturn_postgres() {
     let mut srvc = indexer_service_postgres().await;
     let mut manifest: Manifest =
         serde_yaml::from_str(assets::FUEL_INDEXER_TEST_MANIFEST).expect("Bad yaml file.");
+    manifest.contract_id = Some(get_test_contract_id().to_string());
```

Comment:
@deekerno 
- Maybe we can tuck this into `update_test_manifest_asset_paths` ?
- We technically don't need to set the contract_id but for parity (given the last comment I left about including the contract_id) we can keep it
- `update_test_manifest_asset_paths` is a hack to update the manifest, so might as well tuck this there (as one line) rather than spreading it out. Thoughts? 🧠